gcn_prot/data/protien_graph.py

```diff
@@ -31,7 +30,7 @@ def __init__(self, data, nb_nodes=185, task_type="classification", nb_classes=2)
         self.nb_classes = nb_classes
         self.task_type = task_type
 
-        self.ident = np.eye(nb_nodes)
+        self.ident = np.exp(nb_nodes)
```

Comment:
Why? Take a look at https://docs.scipy.org/doc/numpy/reference/generated/numpy.eye.html. Here, a identity matrix is produced from the size of the aminoacids to then build the mask (so 1 are substituted with 0 in the diagonal if the atom was missing or not well resoluted by the experimental method).